winpr/libwinpr/input/keycode.c

```diff
@@ -450,6 +477,7 @@ DWORD KEYCODE_TO_VKCODE_EVDEV[256] =
 	0, /* <PAST> 143 */
 	0, /* <FIND> 144 */
 	0, /* <CUT> 145 */
+#endif
```

Comment:
One more remark. 137-145 are the same if I haven't missed anything. So why not just have 136 in the ifdef/else?